packages/tabs/components/Tabs.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from "react";
+import { Tabs as ReactTabs, TabList, TabPanel } from "react-tabs";
+import { injectGlobal } from "emotion";
+
+import { TabItemProps } from "./TabItem";
+import { TabTitle } from "..";
+
+// Copy & paste from node_modules/react-tabs/style/react-tabs.css
+// This is needed to give the tabs a style
+injectGlobal`
```

Comment:
This is being flagged by tslint:
```
packages/tabs/components/Tabs.tsx[10, 1]: unused expression, expected an assignment or function call
```

Maybe we should make it a function and call it in the `render` method of `Tabs`?

Reply:
Na, I am going to ignore it for tslint calling this on render could be a huge performance loss. Since we might change the DOM. on every call.